python/TestHarness/testers/CSVDiff.py

```diff
@@ -24,6 +27,13 @@ def __init__(self, name, params):
     def getOutputFiles(self):
         return self.specs['csvdiff']
 
+    # Check that override parameter lists are the same length
+    def checkRunnable(self, options):
+        if (len(self.specs['override_columns']) != len(self.specs['override_rel_err'])) or (len(self.specs['override_columns']) != len(self.specs['override_abs_zero'])) or (len(self.specs['override_rel_err']) != len(self.specs['override_abs_zero'])):
+           self.addCaveats('Override inputs not the same length')
+           self.setStatus(self.bucket_skip.status, self.bucket_fail)
```

Comment:
Do you mean to skip this test and supply a caveat as to the reason why? Or fail it?

Fail method:
```python
self.setStatus('Override inputs not the same length', self.bucket_fail)
```

Skip method:
```python
self.addCaveats('Override inputs not the same length')
self.setStatus(self.bucket_skip.status, self.bucket_skip)
```